### DISCUSSION TITLE
budibase DB tables (cannot modify columns once created)

### DISCUSSION THREAD

**Discussion by SpiceIslander** [Budibase/budibase#9534] [Help] [answered]
Just started trying out budibase and immediately have run into a problem.
I created a table with a couple columns, and can even add rows to the table.
But what I cannot do is change anything about the columns.
I cannot change the 'name', I cannot change the 'type', and cannot change the 'use as table display column' flag.
If the column is created as type 'option', I then cannot change the list of options.
So either I am missing something fundamental, or maybe there is some kind of bug.

**Comment by skorphil** [comment]
Experience same bug https://github.com/Budibase/budibase/discussions/9538
Also, this bug exists even when working with external Mysql db

I personally use sort of a workaround:
I use an external mysql db and when i need to edit a column in budibase, i create a new column in budibase with necessary options and then copy the content of column(i needed to edit) to that new column using external tools like phpmyadmin and sql querry
```
UPDATE `table` SET new_column=orig_column
```  
and then delete the original column

I'm waiting for this bug to be fixed cuz its essential functionality in rapid development

**Comment by skorphil** [comment, marked as answer]
Seems like budibase team fixed this bug

**Reply by melohagan** [reply]
It has! Update to the latest **2.3.12**
